config/environments/production.rb

```diff
@@ -16,7 +16,7 @@
   # config.action_dispatch.rack_cache = true
 
   # Disable Rails's static asset server (Apache or nginx will already do this).
```

Comment:
I love comments that are completely opposite to the actual line below it ...

Reply:
Yeah, I looked for the history of that line, but it was already out of sync with the comment when originally committed by Ohad:

https://github.com/theforeman/foreman/commit/017e1049b87d914af69a8eba8ce18720b1304357#diff-da60b4e96eff2b132991226d308949e23f4ef3aad45ad59edd09cbc32cc6251e